.github/workflows/ci.yml

```diff
@@ -863,14 +863,6 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 2
-      - name: Cache conda
```

Comment:
Since there is no visible result from the step and we do not save this folder with the cache, this step can be deleted.

Reply:
We should then remove this step in push.yml as well.